### PR TITLE
Messidagod-sb

### PR DESCRIPTION
**Pull request by MessiDaGod**
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)

### CODE REVIEW

**Comment by brianamarie** [comment]
@MessiDaGod  There are still a few errors in your file that are keeping the tests from passing. I think it has to do with the capitalization of `latitude` and `longitude`, and an extra file that needs to be deleted.

[Click "Help Me Troubleshoot" for some specific ideas](https://services.github.com/on-demand/intro-to-github/edit-files-on-github), or [watch this video to get some help](https://vimeo.com/201011288). If you need or want help, let me know! If this hasn't been resolved by the end of the week, I'll go ahead and make the changes for you so that it can be merged in. 😄

File: _pins/messidagod-sb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: messidagod-sb
```
Comment (paand) [inline _pins/messidagod-sb]: Wrong username. Change to `githubHandle: MessiDaGod`

File: _pins/messidagod-sb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Latitude: 34.4386
+Longitude: -119.7696
+---
```
Comment (paand) [inline _pins/messidagod-sb]: dd

File: _pins/messidagod-sb.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: messidagod-sb
+Latitude: 34.4386
```
Comment (paand) [inline _pins/messidagod-sb.json]: Capital "L". Spelling should be in lower case i.e. `latitude`

File: _pins/messidagod-sb (hunk 2 above)
Comment (paand) [inline _pins/messidagod-sb]: test

File: _pins/messidagod-sb (hunk 1 above)
Comment (paand) [inline _pins/messidagod-sb]: Wrong username. Should be `MessiDaGod`

File: _pins/messidagod-sb [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+githubHandle: messidagod-sb
+Latitude: 34.4386
```
Comment (paand) [inline _pins/messidagod-sb]: Capital "L". Spelling should be in lower case i.e. `latitude`

File: _pins/messidagod-sb [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+Longitude: -119.7696
```
Comment (paand) [inline _pins/messidagod-sb]: Same problem as the line above. Should be lowercase: `longitude`

**Pull request closed without merging**